perception/elevation_map_loader/src/elevation_map_loader_node.cpp

```diff
@@ -215,9 +216,25 @@ void ElevationMapLoaderNode::inpaintElevationMap(const float radius)
   elevation_map_.add("inpaint_mask", 0.0);
 
   elevation_map_.setBasicLayers(std::vector<std::string>());
-  for (grid_map::GridMapIterator iterator(elevation_map_); !iterator.isPastEnd(); ++iterator) {
-    if (!elevation_map_.isValid(*iterator, layer_name_)) {
-      elevation_map_.at("inpaint_mask", *iterator) = 1.0;
+  if (lane_filter_.use_lane_filter_) {
+    for (const auto & lanelet : lane_filter_.road_lanelets_) {
+      auto lane_polygon = lanelet.polygon2d().basicPolygon();
+      grid_map::Polygon polygon;
+      for (const auto & p : lane_polygon) {
+        polygon.addVertex(grid_map::Position(p[0], p[1]));
+      }
+      for (grid_map::PolygonIterator iterator(elevation_map_, polygon); !iterator.isPastEnd();
```

Comment:
```suggestion
      for (grid_map_utils::PolygonIterator iterator(elevation_map_, polygon); !iterator.isPastEnd();
```

Reply:
@maxime-clem Thanks for your advice!
fixed https://github.com/autowarefoundation/autoware.universe/pull/2885/commits/666caa1403396967ffe605f822a27e100510ae9c